_opcodes/op013.html

```diff
@@ -16,14 +16,80 @@
+  </ul>
 <br/>1   Burning death
+  <ul>
+    <li>Sets STATE[0x00000200][STATE_FLAME_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Character Tint Solid, Character Color, ~[0x00404040] (Darkens avatar)</li>
+    <li>Kicked from Party, Drops Inventory</li>
+  </ul>
 <br/>2   Crushed death
+  <ul>
+    <li>Sets STATE[0x00000100][STATE_EXPLODING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Kicked from Party, Drops Inventory</li>
+  </ul>
 <br/>4   Normal death
+  <ul>
+    <li>Sets STATE[0x00000800][STATE_DEAD]</li>
+    <li>Drops Inventory</li>
+  </ul>
 <br/>8   Chunked death
+  <ul>
+    <li>Sets STATE[0x00000100][STATE_EXPLODING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Sets ANIMATE[0x230][EXPLODING_TORSO]</li>
+    <li>Plays sound ("GORE.WAV")/("GORE2.WAV")</li>
+    <li>Kicked from Party, Drops Inventory, Sets all Color indexes to 70</li>
+  </ul>
 <br/>16  Stoned death
+  <ul>
+    <li>Sets STATE[0x00000080][STATE_STONE_DEATH]</li>
+    <li>Any hostile action against the creature(s) applies an Exploding Stoned Death to them</li>
+    <li>Kicked from Party, Retains Inventory, Overlay's Color Index 72</li>
+  </ul>
 <br/>32  Freezing death
-<br/>64  Exploding stoned death
+  <ul>
+    <li>Applies Exploding Freezing Death to the target with a (random) 0-5 second delay</li>
+    <li>Sets STATE[0x00000040][STATE_FROZEN_DEATH]</li>
+    <li>Any hostile action against the creature applies an Exploding Freezing Death to the target</li>
+    <li>Kicked from Party, Retains Inventory, Overlay's Color Index 71</li>
+  </ul>
+<br/>64  Exploding stoned death/Level Drain death
+  <ul>
+    <li>Sets STATE[0x00000100][STATE_EXPLODING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Sets ANIMATE[0x100][CHUNKS]</li>
+    <li>Kicked from Party, Retains Inventory, Sets all Color indexes to 72</li>
+  </ul>
 <br/>128 Exploding freezing death
-<br/>256 Electrified death 
+  <ul>
+    <li>Sets STATE[0x00000100][STATE_EXPLODING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Sets ANIMATE[0x100][CHUNKS]</li>
+    <li>Kicked from Party, Retains Inventory, Sets all Color indexes to 71</li>
+  </ul>
+<br/>256 Electrified death
+  <ul>
+    <li>Sets STATE[0x00000200][STATE_FLAMING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Character Tint Solid, Character Color, ~[0x00404040] (Darkens avatar)</li>
+    <li>Kicked from Party, Drops Inventory</li>
+  </ul>
 <br/>512 Disintegration death
+  <ul>
+    <li>Sets STATE[0x00000100][STATE_EXPLODING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Sets ANIMATE[0x100][CHUNKS]</li>
+    <li>Kicked from Party, Retains Inventory, Sets all Color indexes to 72</li>
+  </ul>
 <br/>1024 Destruction (in IWD2)
+  <ul>
+    <li>Sets STATE[0x00000100][STATE_EXPLODING_DEATH], STATE[0x00000800][STATE_DEAD]</li>
+    <li>Sets ANIMATE[0x230][EXPLODING_TORSO]</li>
+    <li>Plays sound ("GORE.WAV")/("GORE2.WAV")</li>
+    <li>Kicked from Party, Retains Inventory, Sets all Color indexes to 70</li>
```

Comment:
This one would also be much cleaner through linking to state.ids, so you can then just use eg. STATE_EXPLODING_DEATH. Also, why the funky capitalization when listing features?

Reply:
> Also, why the funky capitalization when listing features?

Simply because they're capitalized in STATE.ids...

Reply:
I meant the last list item.

Reply:
> I meant the last list item.

Do you mean this one?
```
<br/>Other
  <ul>
    <li>Removes all death states, but does not restore HP</li>
    <li>Drops Inventory</li>
  </ul>
```
It's not capitalized (aside from HP...)

Reply:
> <li>Kicked from Party, Retains Inventory, Sets all Color indexes to 70</li>